discovery_engine/lib/src/infrastructure/repository/hive_source_reacted_repo.dart

```diff
@@ -33,6 +33,27 @@ class HiveSourceReactedRepository implements SourceReactedRepository {
   Future<List<SourceReacted>> fetchByReaction(bool like) async =>
       box.values.where((source) => source.liked == like).toList();
 
+  @override
+  Future<SourceReacted?> fetchMinWeightLiked() async {
```

Comment:
❓ since the name is also used for the reduced variable, wdyt about naming the function in the same way? this would also better align the naming with the other new function `fetchOldestDisliked`.
```suggestion
  Future<SourceReacted?> fetchLightestLiked() async {
```

Reply:
it would indeed, sure.